pyproject.toml

```diff
@@ -45,16 +45,18 @@ chgnet = {version = "0.3.8", optional = true}
 dgl = { version = "2.1.0", optional = true } # Pin due to matgl installation issues
 matgl = { version = "1.1.3", optional = true}
 sevenn = { version = "0.9.3", optional = true }
+orb_models = { version ="0.3.0", optional = true }
 torchdata = {version = "0.7.1", optional = true} # Pin due to dgl issue
 torch_geometric = { version = "^2.5.3", optional = true }
 ruff = "^0.5.7"
 
 [tool.poetry.extras]
-all = ["alignn", "chgnet", "matgl", "dgl", "torchdata", "sevenn", "torch_geometric"]
+all = ["alignn", "chgnet", "matgl", "dgl", "torchdata", "sevenn", "torch_geometric", "orb"]
 alignn = ["alignn"]
 chgnet = ["chgnet"]
 m3gnet = ["matgl", "dgl", "torchdata"]
 sevennet = ["sevenn", "torch_geometric"]
+orb = ["orb_models"]
```

Comment:
Do we not need to add [Pynanoflann](https://github.com/orbital-materials/orb-models?tab=readme-ov-file#install) too?